bids/ext/reports/cli.py

```diff
@@ -92,24 +102,30 @@ def set_verbosity(verbosity: int | list[int]) -> None:
         LOGGER.setLevel("DEBUG")
 
 
-def cli(argv: Sequence[str] = sys.argv) -> None:
+def cli(args: Sequence[str] = None, namespace=None) -> None:
     """Entry point."""
     parser = base_parser()
+    opts = parser.parse_args(args, namespace)
 
-    args, unknowns = parser.parse_known_args(argv[1:])
-
-    bids_dir = Path(args.bids_dir[0]).resolve()
-    # output_dir = Path(args.output_dir[0])
-    participant_label = args.participant_label or None
+    bids_dir = opts.bids_dir.resolve()
+    output_dir = opts.output_dir.resolve()
```

Comment:
unrelated: maybe better to use absolute rather than resolve

I have had datalad cases with symbolink links where resolve could lead to a whole bunch of trouble

Reply:
Oh, great point!